src/Http/Http.Extensions/gen/RequestDelegateGeneratorSources.cs

```diff
@@ -485,25 +486,19 @@ public override bool IsDefined(Type attributeType, bool inherit)
     }
 """;
 
-    public static string GetGeneratedRouteBuilderExtensionsSource(string genericThunks, string thunks, string endpoints, string helperMethods, string helperTypes) => $$"""
+    public static string GetGeneratedRouteBuilderExtensionsSource(string endpoints, string helperMethods, string helperTypes) => $$"""
 {{SourceHeader}}
 
-namespace Microsoft.AspNetCore.Builder
+namespace System.Runtime.CompilerServices
 {
     {{GeneratedCodeAttribute}}
-    internal sealed class SourceKey
+    [AttributeUsage(AttributeTargets.Method, AllowMultiple = true)]
+    file sealed class InterceptsLocationAttribute : Attribute
```

Comment:
No public API is included as part of the interceptors feature in preview so we have to define this type ourselves. We use `AllowMultiple = true` here so we can use the same interceptor for `Map` action calls that implement the same signature.